style.css

```diff
@@ -0,0 +1,39 @@
+*{
+    margin:0;
+    padding: 0;
+    box-sizing:border-box;
+}
+
+body {
+    height: 100vh;
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    align-items:center;
+    background: linear-gradient(rgb(48, 213, 232), rgb(128, 141, 215));
+    font-family: Arial, Helvetica, sans-serif;
+    color:antiquewhite;
+}
+
+.location, .temperature {
+    height: 25vh;
+    width: 50%;
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+}
+.orange{
```

Comment:
Class names like these are fine, but in general, try to use class names that convey a sense of the element's role rather than being directly related to its appearance. Here, it's not too big of an issue, since they really don't have much of any other role.